internal/annotations/prefix_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package annotations
+
+import (
+	"testing"
+)
+
+func TestSplitPrefix(t *testing.T) {
+	cases := []struct {
+		prefix    string
+		result    []string
+		wantError bool
+	}{
+		{"", nil, true},
+		{"+operator-sdk", nil, true},
+		{"+operator-sdk:", nil, true},
+		{"+operator-sdk:foo", []string{"foo"}, false},
+		{"+operator-sdk:foo:bar", []string{"foo", "bar"}, false},
+	}
+
+	for i, c := range cases {
+		got, err := SplitPrefix(c.prefix)
```

Comment:
Maybe this is a style nit but can we run each case as a named subtest so it's more clear which test case failed?
Right now it's the index `i` of the test case I believe:
```Go
t.Errorf("Case %d: wanted result %+q, got error: %v", i, c.result, err)
```
The name can just be the input being tested i.e prefix.
```Go
for _, c := range cases {
    t.Run(c.prefix, func(t *testing.T) {
        got, err := SplitPrefix(c.prefix)
        ...
    }
```

I think we already do this in other unit tests but it would be good to standardize these [table style tests](https://github.com/golang/go/wiki/TableDrivenTests).
https://github.com/operator-framework/operator-sdk/blob/master/internal/pkg/scaffold/helm/chart_test.go#L248-L251

Reply:
Yeah that's more informative. I'll switch up the style.